source/includes/cli-guide/_install.html.md

```diff
@@ -202,7 +217,9 @@ python mountaincar_simulator.py --train-brain=myMountainCarBrain --headless
 python3 mountaincar_simulator.py --train-brain=myMountainCarBrain --headless
 ```
 
-Everything is in place, it's time to start the simulator and test things out. To do this, call Python and then the simulator file. Training will begin automatically and if the simulator gets disconnected, training resumes from the same point when the simulator is reconnected if it's within an hour of the disconnect. Remember, as mentioned in [What is a simulator?][7] if you want your training to take place overnight, make sure that your computer won't go to sleep and disconnect for more than an hour or that training time will be lost!
+Start the training mode for your BRAIN with [`bonsai train start`][20].
```

Comment:
`the training mode` doesn't sound right to me - We're using the word state in the response to `bonsai train status`.  This could say `Starting training your brain` maybe?